app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -113,6 +114,11 @@ data class AppSettings(
         defaultValue = DEFAULT_HIDE_LETTERS.toString(),
     )
     val hideLetters: Int,
+    @ColumnInfo(
+        name = "hide_symbols",
+        defaultValue = DEFAULT_HIDE_SYMBOLS.toString(),
+    )
+    val hideSymbols: Int,
```

Comment:
Should go at the bottom, settings always are appended as columns.

Reply:
You also need to increase the DB version and add a DB migration. Look below in this file for how that's done.

Reply:
Done